### PR TITLE
Update .gitmodules

### PR DESCRIPTION
**Pull request by ttgb**
Modify the libwebp url to github mirror, as many people in China can not access googlesource because the GFW

### CODE REVIEW

**Comment by mythodeia** [comment]
@bpoplauschi looks good.

**Comment by bpoplauschi** [comment]
Looks good, not sure what the impact is on the existing users and if this doesn't break anything, but we'll leave it as it is (merged) and keep an eye on it. Thanks guys

**Comment by ttgb** [comment]
I already test this with the scenario you said: I first pod install the origin libwepp with VPN，and then change the new spec file I modified, It works just as I expected. so this change should not impact any existing users.

**Comment by bpoplauschi** [comment]
:+1:

**Comment by mrxiaobin** [comment]
still like that:
 ......unable to access 'https://chromium.googlesource.com/webm/libwebp/': Could not resolve host: chromium.googlesource.com

**Comment by mythodeia** [comment]
@mrxiaobin after this pull the url is now https://github.com/webmproject/libwebp not the one you wrote

**Comment by mrxiaobin** [comment]
@mythodeia but i'm using cocoapods, using " pod 'SDWebImage/WebP' " int the podfile

 I tried the shadowsocks proxy, and i can open the website"https://chromium.googlesource.com/webm/libwebp/"

but when i use pod install,still shown the error:"Cloning into '/var/folders/1t/40bhqkc92wg8th1lbnp7z_j80000gn/T/d20160521-1465-15yvxye'...
fatal: unable to access 'https://chromium.googlesource.com/webm/libwebp/': Failed to connect to chromium.googlesource.com port 443: Operation timed out".

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "Vendors/libwebp"]
 	path = Vendors/libwebp
-	url = https://chromium.googlesource.com/webm/libwebp
+	url = https://github.com/webmproject/libwebp
```
Comment (qianqichao) [inline .gitmodules]: https://github.com/webmproject/libwebp

**Comment by codefei** [comment]
It seems libwebp is still access 'https://chromium.googlesource.com/webm/libwebp/' not 'https://github.com/webmproject/libwebp'

**Comment by mengkeer** [comment]
It seems libwebp is still access 'https://chromium.googlesource.com/webm/libwebp/'   , can you fix it?  or can you provide a solution？

**Comment by bpoplauschi** [comment]
You probably have an older version because all our latest versions use https://github.com/webmproject/libwebp

**Pull request merged**